src/components/Filter/modules/MultiSelectFilter/multiSelectUtil.ts

```diff
@@ -17,3 +17,7 @@ export const parseItems = (item: string, delimiter?: string): string[] => {
         .map((a) => a.trim())
         .filter((a) => a);
 };
+
+export const getUniqueOptions = (itemsStateCopy: MultiSelectOption[]): MultiSelectOption[] => {
+    return itemsStateCopy.filter((unique, index) => unique.value.indexOf(unique.value) !== index);
```

Comment:
This code doesn't quite work expected. You can check by changing the `itemsStateMock` in the test to look like the following:
```ts
const itemsStateMock =
            [
                { 'label': 'colors', 'value': 'colors' },
                { 'label': 'sizes', 'value': 'sizes' },
                { 'label': 'shapes', 'value': 'shapes' },
                { 'label': 'colors', 'value': 'colors' },
                { 'label': 'colors', 'value': 'colors' }
            ];
```

I think instead of doing `unique.value.indexOf` (which checks if the provided string appears in the value), we instead need to be building a new array and checking if there is already an item in that array with the same value.